fails:Kernel#open is a private method
fails:Kernel#open calls #to_open on argument
fails:An exception occurred during: before :each
fails:Kernel#open opens a file when given a valid filename
